sympy/functions/combinatorial/tests/test_comb_numbers.py

```diff
@@ -103,6 +103,10 @@ def test_bell():
     m = Symbol('m', integer=True)
     assert bell(-1).evalf() == bell(m).rewrite(Sum).evalf(subs={m: -1})
 
+    # issue 9184
+    n = Dummy('n')
+    assert bell(n).limit(n, S.Infinity) == S.Infinity
```

Comment:
Yes. Though the issue that this fixes (#9184) is using a limit, so it's nice to check that it actually works.